packages/ui/src/components/va-avatar-group/VaAvatarGroup.vue

```diff
@@ -9,11 +9,11 @@
       :key="idx"
       v-bind="{ ...avatarProps, ...option }"
       role="listitem"
-      tabindex="0"
     />
     <slot name="rest" v-bind="avatarProps">
       <va-avatar
         v-bind="avatarProps"
+        color="secondary"
```

Comment:
@rustem-nasyrov, do we always want to use the `secondary` colour here? Maybe, use `color` prop instead?

Reply:
Nice catch. We need to pass here avatarProps and we want color to be secondary by default, so we need to redeclare color prop in definition.

Reply:
I added just a new prop for that... It might be solved with css variables, but we don't usually do that.